inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/infer_request/infer_requset_dynamic.cpp

```diff
@@ -24,6 +24,7 @@ std::shared_ptr<ngraph::Function> getFunction1() {
 
     auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
     params.front()->set_friendly_name("Param_1");
+    params.front()->get_output_tensor(0).set_names({"Tensor_1"});
```

Comment:
`Tensor_1` looks like auto generated name, we can have a collision in the future. I propose to use another name.

Reply:
used can provide any names, we should not avoid them :)